lang/he.json

```diff
@@ -0,0 +1,84 @@
+{
+	"Audio Player": "נַגָּן שמע",
+	"Video Player": "נַגָּן וידאו",
+	"Play": "נַגֵּן",
+	"Pause": "השהה",
+	"Replay": "נַגֵּן שוב",
+	"Current Time": "זמן נוכחי",
+	"Duration Time": "זמן כולל",
+	"Remaining Time": "זמן נותר",
+	"Stream Type": "סוג Stream",
+	"LIVE": "שידור חי",
+	"Loaded": "נטען",
+	"Progress": "התקדמות",
+	"Progress Bar": "סרגל התקדמות",
+	"progress bar timing: currentTime={1} duration={2}": "{1} מתוך {2}}",
```

Comment:
Is this malformed or just rendering issues? Maybe there's just an extra `}` at the end/beginning?

Reply:
I think it only appears weird because Hebrew is RTL and numbers are LTR.
Ill give it another test run to make sure it appears properly in the player itself and get back to you.